arraycontext/impl/jax/__init__.py

```diff
@@ -54,8 +54,8 @@ class and performs all array operations eagerly. See
     def __init__(self) -> None:
         super().__init__()
 
-        from jax.numpy import DeviceArray
-        self.array_types = (DeviceArray, )
+        from jax import Array
+        self.array_types = (Array, )
```

Comment:
Can this use `jnp.ndarray`? From what I know that aliases to the correct underlying class on all versions.

Reply:
Both `jax.numpy.ndarray` and `jax.Array` seem to work. I'm not sure which one is better.

Reply:
`jnp.ndarray` should alias to `DeviceArray` on older versions and `Array` on newer versions of jax. I guess it depends on what versions we want to support?

Reply:
That's a good point, thanks! I changed it to `jnp.ndarray` in [45482ab](https://github.com/inducer/arraycontext/pull/211/commits/45482ab962934c5a6755412ea45b397c395fcdb8).